pkg/daemon/update.go

```diff
@@ -1287,9 +1287,20 @@ func (dn *Daemon) writeFiles(files []ign3types.File) error {
 	for _, file := range files {
 		glog.Infof("Writing file %q", file.Path)
 
-		contents, err := dataurl.DecodeString(*file.Contents.Source)
-		if err != nil {
-			return err
+		// We don't support appends in the file section, so instead of waiting to fail validation,
+		// let's explicitly fail here.
+		if len(file.Append) > 0 {
+			return fmt.Errorf("found an append section when writing files. Append is not supported")
```

Comment:
to not regress in 4.6 we should just move this to a warning right?

Reply:
See below, I don't think this should have ever worked. Just moving error detection.